File: apps/web/config/product.ts
/**
 * Product identity — the frontend half of the three-file rename surface.
 * Mirrors packages/config/nemonis_config/product.py.
 */

export const PRODUCT_NAME = "Ñemonis";
export const PRODUCT_SLUG = "nemonis";
export const TAGLINE = "Forex research, backtesting and risk-control platform";

export const SAFETY_NOTICE =
  "Research and paper-trading only. This build cannot place real-money orders: no broker adapter exists.";

export const API_URL =
  process.env.NEXT_PUBLIC_NEMONIS_API_URL ?? "http://localhost:8787";

File: packages/config/meridian_config/product.py
"""Product identity.

One of only three places the product name appears (the others being
``apps/web/config/product.ts`` and the ``MERIDIAN_`` environment prefix).
Renaming the product is an edit to these three, not a codebase-wide search.
"""

from __future__ import annotations

PRODUCT_NAME = "Meridian"
PRODUCT_SLUG = "meridian"
ENV_PREFIX = "MERIDIAN_"
VERSION = "0.1.0"

TAGLINE = "Forex research, backtesting and risk-control platform"

# Shown wherever the user could mistake this for a live trading system.
SAFETY_NOTICE = (
    "Research and paper-trading only. This build cannot place real-money orders: "
    "no broker adapter exists."
)
